includes/migration-guide/runtime/wcf/replace-method-odata-urls-disabled-by-default.md

```diff
@@ -12,8 +12,16 @@ If the replace method is required (which is uncommon), it can be re-enabled thro
 |:--------|:------------|
 | Scope   |Edge|
 |Version|4.5|
-|Type|Runtime
+|Type|Runtime|
 
 #### Affected APIs
 
--<xref:System.Data.Services.DataService%601?displayProperty=nameWithType></li></ul>|
+- <xref:System.Data.Services.DataService%601?displayProperty=nameWithType>
+
+<!--
+
+#### Affected APIs
+
+- `T:System.Data.Services.DataService`1`
```

Comment:
I think you need to use two backticks around the code block when it contains a backtick.

```suggestion
- ``T:System.Data.Services.DataService`1``
```

Reply:
There are numerous other places to fix that too.